e2e/tests/datasets/dataset-create.spec.ts

```diff
@@ -9,8 +9,8 @@ import WorkspaceAboutPage from 'app/page/workspace-about-page';
 import { config } from 'resources/workbench-config';
 import { makeWorkspaceName } from 'utils/str-utils';
 
-// 30 minutes. Test involves notebook that could take a long time to create.
```

Comment:
What is the reason to refuce this? Does notebook creation becomes faster?

Reply:
Comment was an error. None of tests in this file create or start a notebook.